mevislab.github.io/content/tutorials/summary/summary8.md

```diff
@@ -0,0 +1,327 @@
+    Field out2D { internalName = RemoteRendering2D.output }
+    Field out3D { internalName = RemoteRendering3D.output }
+    Field outSegmentationMask { internalName = CloseGap.output0 }
+  }
+  Parameters {
+    Field openFile {
+      type         = String
+      internalName = LocalImage.name
+    }
+    Field selectOverlayColor {
+      internalName = SoView2DOverlay.baseColor
+      type         = Color
+    }
+    Field selectOverlayTransparency {
+      internalName = SoView2DOverlay.alphaFactor
+    }
+    Field imageAlpha {
+      internalName = SoWEMRendererImage.faceAlphaValue
+      type         = Integer
+      min          = 0
+      max          = 1
+    }
+    Field thresholdInterval {
+      internalName = RegionGrowing.autoThresholdIntervalSizeInPercent
+      type         = Integer
+      min          = 0
+      max          = 100
+    }
+    Field isoValueImage  {
+      internalName = IsoSurfaceImage.isoValue
+      type         = Integer
+      min          = 0
+      max          = 1000
+    }
+    Field selected3DView {
+      type   = Enum
+      values = Segmented,File,Both
+    }
+    Field totalVolume {
+      internalName = CalculateVolume.totalVolume
+      editable     = False
+    }
+    Field resetApplication {
+      type  = Trigger
+      title = Reset
+    }
+    Field markerPosition {
+      type         = Vector3
+    }
+    Field applyMarker {
+      type  = Trigger
+      title = Add
+    }
+  }
+}
+```
+{{</highlight>}}
+
+Reloading your web macro in MeVisLab SDK now shows the same outputs as the original macro module. The only difference is the type of your output. It changed from MLImage and Inventor Scene to MLBase from your `RemoteRendering` modules.
+
+![Macro Modules](/images/tutorials/summary/Example8_7.png "Macro Modules")
+
+The internal network of your web macro should look like this:
+
+![Macro Modules](/images/tutorials/summary/Example8_8.png "Macro Modules")
+
+You can emulate the final viewer by adding a `RemoteRenderingClient` module to the outputs of your web macro.
+
+![RemoteRenderingClient](/images/tutorials/summary/Example8_9.png "RemoteRenderingClient")
+
+Open the *\*.script* files of your macro modules and copy the *FieldListeners* from *Commands* section of your *TutorialSummary.script* to *TutorialSummaryBrowser.script*.
+
+{{< highlight filename="TutorialSummaryBrowser.script" >}}
+```Stan
+Commands {
+  source = $(LOCAL)/TutorialSummaryBrowser.py
+  
+  FieldListener selected3DView {
+    command = viewSelectionChanged
+  }
+  FieldListener resetApplication {
+    command = resetApplication
+  }
+  FieldListener markerPosition {
+    command = insertPosition
+  }
+  FieldListener applyMarker {
+    command = applyPosition
+  }
+}
+```
+{{</highlight>}}
+
+Also copy the *Window* section to your web macro module. The *Box* of the *Viewing* tab needs to be modified because we are not using the `View3D` and `SoExaminerViewer` outputs anymore but the `RemoteRendering` outputs.
```

Comment:
because we are now using the `RemoteRendering` outputs instead of the `View3D` and `SoExaminerViewer` outputs